test/bigtable/index.js

```diff
@@ -235,7 +229,7 @@ describe('Bigtable', function() {
       bigtable.createTable(TABLE_ID, function(err, table, apiResponse) {
         assert.ifError(err);
         assert(table instanceof FakeTable);
```

Comment:
This should be testing that `bigtable.table` was called correctly and that its return value was used.